react/Video.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import { VideoPlayer } from './interfaces'
+import VimeoPlayer from './players/VimeoPlayer'
+import HTML5Player from './players/HTML5Player'
+import YoutubePlayer from './players/YoutubePlayer'
+
+const Video: StorefrontFunctionComponent<VideoPlayer> = (props) => {
+  const { src, name, description, poster, uploadDate } = props
+  const isVimeo = /vimeo/.test(src)
+  const isYoutube = /youtube|youtu.be/.test(src)
```

Comment:
We don't need to create a new RegExp at every render
```suggestion
const VIMEO_PATTERN = /vimeo/
const YOUTUBE_PATTERN = /youtube|youtu.be/

const Video: StorefrontFunctionComponent<VideoPlayer> = (props) => {
  const { src, name, description, poster, uploadDate } = props
  const isVimeo = VIMEO_PATTERN.test(src)
  const isYoutube = YOUTUBE_PATTERN.test(src)
```